plugin.go

```diff
@@ -22,11 +24,11 @@ func (p *MatterpollPlugin) OnActivate(api plugin.API) error {
 
 func (p *MatterpollPlugin) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
 	input := ParseInput(args.Command)
-	if len(input) == 0 {
+	if len(input) < 3 {
```

Comment:
I suppose that there may be a poll having just one answer.
e.g.) "Let's have a lunch together" "Yes!"

So I think that the number here had better to be `2`, not `3`,

Reply:
:+1: